forge/comms/v2AuthRoutes.js

```diff
@@ -0,0 +1,144 @@
+const mqttMatch = require('mqtt-match')
+
+module.exports = async function (app) {
+    app.post('/auth', {
+        schema: {
+            body: {
+                type: 'object',
+                required: ['username', 'password'],
+                properties: {
+                    username: { type: 'string' },
+                    password: { type: 'string' },
+                    clientId: { type: 'string' }
+                }
+            },
+            response: {
+                200: {
+                    type: 'object',
+                    required: ['result'],
+                    properties: {
+                        result: { type: 'string' },
+                        is_superuser: { type: 'boolean' },
+                        client_attrs: { type: 'object', additionalProperties: true }
+                    },
+                    additionalProperties: true
+                }
+            }
+        }
+    }, async (request, reply) => {
+        const username = request.body.username
+        const password = request.body.password
+        const clientId = request.body.clientId
+        if ((username.startsWith('device:') && password.startsWith('ffbd_')) ||
+            (username.startsWith('project:') && password.startsWith('ffbp_')) ||
+            (username.startsWith('frontend:') && password.startsWith('ffbf_')) ||
+            (username === 'forge_platform')) {
+            const isValid = await app.db.controllers.BrokerClient.authenticateCredentials(
+                username,
+                password
+            )
+            if (isValid) {
+                reply.send({
+                    result: 'allow',
+                    is_superuser: false,
+                    client_attrs: {
+                        team: 'ff/v1/internal/c/'
```

Comment:
```suggestion
                        team: ''
```
This should be empty to allow for interaction between project nodes and team broker clients

Reply:
Agreed - I don't think we should be setting the prefix for the existing clients as they already fully specify their topic space with `ff/v1/:teamId/...` - no need to push that under an even deeper hierarchy.